dev-aws/kafka-shared-msk/cbc/cbc.tf

```diff
@@ -185,14 +236,14 @@ module "cbc_fraud_detection_consumer" {
 
 module "cbc_events_indexer" {
   source           = "../../../modules/tls-app"
-  consume_topics   = [kafka_topic.FraudEvents.name, kafka_topic.rating_events_v3.name, kafka_topic.lifecycle_events_v2.name, kafka_topic.charges_events_v1.name, kafka_topic.challenge_events_v1.name, kafka_topic.topup_events_v1.name, kafka_topic.transaction_events_v3.name, kafka_topic.openbanking_events_v1.name, kafka_topic.order_events_v1.name, kafka_topic.paymentology_events_v1.name]
+  consume_topics   = [kafka_topic.FraudEvents.name, kafka_topic.rating_events_v3.name, kafka_topic.lifecycle_events_v2.name, kafka_topic.charges_events_v1.name, kafka_topic.challenge_events_v1.name, kafka_topic.topup_events_v1.name, kafka_topic.transaction_events_v3.name, kafka_topic.openbanking_events_v1.name, kafka_topic.order_events_v1.name, kafka_topic.paymentology_events_v1.name, kafka_topic.sodexo_events_v1.name, kafka_topic.verification_events_v1.name]
```

Comment:
Terraform [supports](https://developer.hashicorp.com/terraform/language/expressions/types#lists-tuples) splitting lists into multiple lines for readability. Might be worth doing.

Reply:
done!